src/config/settings/development.py

```diff
@@ -97,13 +97,14 @@ def monkey_patch():
 # https://docs.djangoproject.com/en/dev/ref/settings/#default-from-email
 DEFAULT_FROM_EMAIL = env(
     "DJANGO_DEFAULT_FROM_EMAIL",
-    default="Republican Antiquarians Research Database <noreply@example.com>",
+    default="Republican Antiquarians Research Database (dev) <noreply@example.com>",
```

Comment:
It shouldn't say "(dev)" in the production version, right? This line shouldn't be merged.

Reply:
This settings file is only used by the development server. The production server uses src/config/settings/production.py which has these variables set as suggested.

Reply:
Ah yes :) Silly me!